### DISCUSSION TITLE
Desconto da OS - escolher entre % ou valor.

### DISCUSSION THREAD

**Discussion by jsfachini** [RamonSilva20/mapos#1761] [Ideas]
Boa Noite, seria possível ter a opção de além da porcentagem poder escolher o valor?
supondo que um produto custe R$450,00 e aplicar 11% de desconto ficaria R$ 400,50 
seria bom ter uma opção de poder escolher o valor total do desconto para finalizar a OS com valor cheio.

**Comment by willph** [comment]
> Boa Noite, seria possível ter a opção de além da porcentagem poder escolher o valor? supondo que um produto custe R$450,00 e aplicar 11% de desconto ficaria R$ 400,50 seria bom ter uma opção de poder escolher o valor total do desconto para finalizar a OS com valor cheio.

O desconto aceita casa decimal. 

11.50......

**Comment by andremp15** [comment]
Oi gostaria de saber se teria como ao cadastrar usuário tirar uma porcentagem para o vendedor ou para o técnico de venda ou serviço
